peopleatschool.rb

```diff
@@ -0,0 +1,35 @@
+class PeopleAtSchool
+ attr_reader :age, :salary, :phase, :target_raise
```

Comment:
Not everyone at the school shares these things.  They all share only these common things:

```ruby
require_relative './hiFiveAble'

class Person
  include HiFiveAble

  attr_reader :age, :phase
  attr_accessor :name

  def initialize(options = {})
    @age = options.fetch(:age, 0)
    @name = options.fetch(:name, "")
  end
end
```